### PR TITLE
2017VLLCards

### PR DESCRIPTION
**Pull request by shchauha**
cards for Vectorlike lepton model. Analysis targetting Moriond.

### CODE REVIEW

**Comment by efeyazgan** [comment]
Please leave one representative file for a single mass point and if possible add a script to generate the other mass points.

**Comment by shchauha** [comment]
Done. 
I removed all mass points except 300 and added script to produce others.

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VectorLikeLeptons/VectorLikeLeptons_M-300_13TeV-madgraph_MLM/VectorLikeLeptons_M-300_13TeV-madgraph_MLM_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+#*                                                          *
+#************************************************************
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model VectorLikeLeptons_M-300_UFO
+# Define multiparticle labels
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VectorLikeLeptons/VectorLikeLeptons_M-300_13TeV-madgraph_MLM/VectorLikeLeptons_M-300_13TeV-madgraph_MLM_proc_card.dat]: please follow https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VectorLikeLeptons/VectorLikeLeptons_M-300_13TeV-madgraph_MLM/VectorLikeLeptons_M-300_13TeV-madgraph_MLM_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+# Store info for systematics studies                                 *
+# WARNING: If use_syst is T, matched Pythia output is                *
+#          meaningful ONLY if plotted taking matchscale              *
+#          reweighting into account!                                 *
+#*********************************************************************
+   True  = use_syst      ! Enable systematics studies
+#
+#**************************************
+# Parameter of the systematics study
+#  will be used by SysCalc (if installed)
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VectorLikeLeptons/VectorLikeLeptons_M-300_13TeV-madgraph_MLM/VectorLikeLeptons_M-300_13TeV-madgraph_MLM_run_card.dat]: please remoe the syscalc part

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VectorLikeLeptons/VectorLikeLeptons_M-300_13TeV-madgraph_MLM/VectorLikeLeptons_M-300_13TeV-madgraph_MLM_proc_card.dat [hunk 3]
```diff
@@ -0,0 +1,44 @@
+#************************************************************
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VectorLikeLeptons/VectorLikeLeptons_M-300_13TeV-madgraph_MLM/VectorLikeLeptons_M-300_13TeV-madgraph_MLM_proc_card.dat]: please remove the banner

**Comment by shchauha** [comment]
Thanks, 
cleaned up the cards using 2017 instructions.

**Pull request merged**